### ISSUE TITLE
[BUG] Automatic workspace switch when a link is clicked

### ISSUE THREAD

**Issue by ekkinox** [pop-os/cosmic-epoch#637]
Let's say I have 2 workspaces:
- workspace 1 with firefox open
- workspace 2 with slack

When I'm on workspace 2, clicking a link from slack, it should auto switch to workspace 1 and put focus on firefox, opening the link.

**Comment by dmyyy** [comment]
Application switching when clicking on links in general. If I click a link in say discord it should switch to my browser.

**Comment by Brandon168** [comment]
Agreed. I keep clicking links 3+ times before remembering to switch workspaces to the browser ... only to find 3+ links opened.

**Comment by relma2** [comment]
Am experiencing the same issue; please fix this devs

**Comment by git-f0x** [comment]
I think this is a duplicate of https://github.com/pop-os/cosmic-comp/issues/192.
